### ISSUE TITLE
Broken on python 3.11

### ISSUE THREAD

**Issue by BloodZero0** [mov-cli/mov-cli#65]
on python 3.11 the program doesn't work, I always get this error:

`Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\onepi\scoop\apps\python\current\Scripts\mov-cli.exe\__main__.py", line 4, in <module>
  File "C:\Users\onepi\scoop\apps\python\current\Lib\site-packages\mov_cli\__init__.py", line 1, in <module>
    from . import __main__
  File "C:\Users\onepi\scoop\apps\python\current\Lib\site-packages\mov_cli\__main__.py", line 6, in <module>
    from .utils.config import config
ModuleNotFoundError: No module named 'mov_cli.utils.config'`

**Comment by R3tr0Ananas** [comment]
What Version did you use the Source or pip version?

**Comment by ghost** [comment]
Should be fixed

**Comment by THEGOLDENPRO** [comment]
I don't think it's been fixed, on the latest release [``0.4.0``](https://pypi.org/project/mov-cli/0.4.0/) I'm getting this...

```python
Traceback (most recent call last):
  File "C:\Users\jassi\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\jassi\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\jassi\AppData\Local\Programs\Python\Python38\Scripts\mov-cli.exe\__main__.py", line 4, in <module>
  File "C:\Users\jassi\AppData\Local\Programs\Python\Python38\lib\site-packages\mov_cli\__init__.py", line 1, in <module>
    from . import __main__
  File "C:\Users\jassi\AppData\Local\Programs\Python\Python38\lib\site-packages\mov_cli\__main__.py", line 57, in <module>
    default=f"{config.getprovider()}",
NameError: name 'config' is not defined
```

**Comment by THEGOLDENPRO** [comment]
👀 For short future reference --> install [`0.3.8`](https://pypi.org/project/mov-cli/0.3.8/) for a temporary fix.

```cmd
pip install mov-cli==0.3.8 -U
```
